connection.js

```diff
@@ -1182,23 +1182,23 @@ class Connection {
         const src_port = match[4];
         const dst_port = match[5];
 
+        // TODO: is it at all possible to avoid the ESLint disable here?
         // Validate source/destination IP
         /*eslint no-fallthrough: 0 */
         switch (proto) {
             case 'TCP4':
                 if (ipaddr.IPv4.isValid(src_ip) && ipaddr.IPv4.isValid(dst_ip)) {
-                    break;
+                    break; // Valid IPv4 address
                 }
             case 'TCP6':
                 if (ipaddr.IPv6.isValid(src_ip) && ipaddr.IPv6.isValid(dst_ip)) {
-                    break;
+                    break; // Valid IPv6 address
```

Comment:
Seriously. `Valid` is a subset of valid ONE LINE UP. `IPv6` is a completely redundant token. How are these comments an improvement?

Reply:
Because I don't have to do mental math on what's going on there. The whole reason I tried to change this is because in my take on readability, it looked like the traffic was getting sidelined in some weird way. It was affirming positives, and then acting on a negative; vs acting on all possible negatives.

Reply:
But your mental math shortcut, and thus your comment, isn't RIGHT. This code validates that both the source and destination IPs are valid IPv6. IMO, because of the use of good library and function names (IPv6.isValid()), the code is self documenting. Any comments thus added should add additional clarity, and certainly not wrong impressions (address, vs addresses).

Reply:
Ultimately, I am trying to resolve https://rules.sonarsource.com/javascript/RSPEC-128 : it is considered a blocker, and a CWE security hazard https://cwe.mitre.org/data/definitions/484 ; it's a "may be right, but won't always be right" kind of situation.